src/routes/costModels/createCostModelWizard/steps.tsx

```diff
@@ -1,20 +1,20 @@
+import { intl } from 'components/i18n';
 import messages from 'locales/messages';
-import { MessageDescriptor } from 'react-intl';
 import { countDecimals, isPercentageFormatValid } from 'utils/format';
 
-export const nameErrors = (name: string): MessageDescriptor | null => {
+export const nameErrors = (name: string): string | null => {
   if (name.length === 0) {
-    return messages.costModelsRequiredField;
+    return intl.formatMessage(messages.costModelsRequiredField);
   }
   if (name.length > 100) {
-    return messages.costModelsInfoTooLong;
+    return intl.formatMessage(messages.costModelsInfoTooLong);
   }
   return null;
 };
 
-export const descriptionErrors = (description: string): MessageDescriptor | null => {
+export const descriptionErrors = (description: string): string | null => {
```

Comment:
I believe the intention here was to return a `MessageDescriptor`, but call `formatMessage` where we already have access to the `i18n` object in `generalInformation.tsx`.

See https://github.com/project-koku/koku-ui/blob/main/src/routes/costModels/createCostModelWizard/generalInformation.tsx#L112